client/nps.js

```diff
@@ -44,12 +44,6 @@ window.addEventListener( 'load', () =>
 
 					window.localStorage.setItem( key, viewCount );
 
-					// eslint-disable-next-line no-console
```

Comment:
Should we also just remove the command? We should do it for the release anyway and the preview allows you to to trigger it manually anyway.